### PR TITLE
Update Bootstrap: Sample QC Tab

### PR DESCRIPTION
**Pull request by maiwald**

### CODE REVIEW

File: app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockCnmr.js [hunk 1]
```diff
@@ -89,23 +88,18 @@ const BlockCnmr = ({ ansCnmr }) => {
...
           </span>
         </div>
-        <Panel
-          className="qc-detail-panel"
-          id="qc-detail-panel-cnmr"
-          defaultExpanded={false}
-        >
-          <Panel.Heading>
-            <Panel.Title className="qc-detail-panel-title" toggle>
+
```
Comment (jhmegorei) [inline app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockCnmr.js:91]: Shouldn't this be a card? It's only a single item.

File: app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockHnmr.js [hunk 2]
```diff
@@ -89,23 +88,18 @@ const BlockHnmr = ({ ansHnmr }) => {
...
           </span>
         </div>
-        <Panel
-          className="qc-detail-panel"
-          id="qc-detail-panel-hnmr"
-          defaultExpanded={false}
-        >
-          <Panel.Heading>
-            <Panel.Title className="qc-detail-panel-title" toggle>
+
```
Comment (jhmegorei) [inline app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockHnmr.js:91]: Same here, only one item -> Card?

File: app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockIr.js [hunk 3]
```diff
@@ -66,23 +65,18 @@ const BlockIr = ({ ansIr }) => {
...
           {iconByMargin(ansOwnF90, 0)}
         </div>
-        <Panel
-          className="qc-detail-panel"
-          id="qc-detail-panel-ir"
-          defaultExpanded={false}
-        >
-          <Panel.Heading>
-            <Panel.Title className="qc-detail-panel-title" toggle>
+
```
Comment (jhmegorei) [inline app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockIr.js:68]: dito

File: app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockCnmr.js (hunk 1 above)
Comment (JoachimTillessen) [reply app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockCnmr.js:91]: If you want the header to collapse and expand the body on click. I would use a single item accordion. It's a pattern that is somewhat special within chemotion, so I think repurposing the accordion component for this is more effective than to invent an interactive variant of the card component.

**Review by JoachimTillessen — APPROVED** [review]

File: app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockCnmr.js (hunk 1 above)
Comment (jhmegorei) [reply app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockCnmr.js:91]: oh ok, I did miss the collapsed part.

**Pull request merged**